id/news/_posts/2020-03-19-json-dos-cve-2020-10663.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: news_post
+title: "CVE-2020-10663: Kerentanan Penciptaan Unsafe Object pada JSON (Perbaikan tambahan)"
+author: "mame"
+translator: "meisyal"
+date: 2020-03-19 13:00:00 +0000
+tags: security
+lang: id
+---
+
+Ada sebuah kerentanan penciptaan *unsafe object* pada *gem* json yang di-*bundle*
+dengan Ruby. Kerentanan ini telah ditetapkan dengan penanda CVE [CVE-2020-10663](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-10663).
+Kami sangat menganjurkan untuk memperbarui *gem* json.
+
+## Detail
+
+Ketika mem-*parsing* suatu dokumen JSON, *gem* json (termasuk yang di-*bundle*
+dengan Ruby) dapat disalahgunakan untuk membuat objek tertentu pada sistem
+yang dituju.
+
+Masalah ini sama dengan [CVE-2013-0269](https://www.ruby-lang.org/en/news/2013/02/22/json-dos-cve-2013-0269/).
+Perbaikan sebelumnya belum selesai, yang membahas `JSON.parse(user_input)`,
+tetapi tidak membahas beberapa gaya JSON *parsing* lainnya, seperti
+`JSON(user_input)` dan `JSON.parse(user_input, nil)`.
+
+Lihat [CVE-2013-0269](https://www.ruby-lang.org/en/news/2013/02/22/json-dos-cve-2013-0269/)
+untuk detail. Catat bahwa masalah ini bisa dieksploitasi yang menyebabkan
+*Denial of Service* dengan membuat banyak objek *garbage-uncollectable Symbol*,
+tetapi penyerangan semacam ini tidak lagi valid karena objeck *Symbol* sekarang
```

Comment:
typo here, should be objek

Reply:
Thanks for the catch @gozali. I have fixed it.